src/perf_tool_backend.ml

```diff
@@ -106,12 +114,18 @@ module Recording = struct
            %!";
         []
     in
+    let snapshot_size_arg =
```

Comment:
Nit: we call these `opt` elsewhere in this fn, let's be consistent one way or another. No preference for arg vs opt.